plugins/system/debug/src/Storage/FileStorage.php

```diff
@@ -89,7 +89,12 @@ public function find(array $filters = [], $max = 20, $offset = 0)
 		usort(
 			$files,
 			function ($a, $b) {
-				return $a['time'] < $b['time'];
+				if ($a['time'] === $b['time'])
+				{
+					return 0;
+				}
+
+				return $a['time'] < $b['time'] ? 1 : -1;
```

Comment:
```suggestion
				return $a['time'] < $b['time'] ? -1 : 1;
```
From https://www.php.net/manual/en/function.usort.php :

> The comparison function must return an integer less than, equal to, or greater than zero if the first argument is considered to be respectively less than, equal to, or greater than the second.

Reply:
`+` or `-` defines the direction, here we need `1 : -1` - descending.
you can test it by changing this line and look how files are sorted in the popup

Reply:
Ok, for descending it has to be like that.